src/www/widgets/widgets/gateways.widget.php

```diff
@@ -111,16 +105,12 @@ function gateways_widget_update(sender, data)
     <table class="table table-condensed">
       <tr>
         <td>
-          <select id="gatewaysinvert" name="gatewaysinvert" class="selectpicker_widget">
-            <option value="" <?= empty($pconfig['gatewaysinvert']) ? 'selected="selected"' : '' ?>><?= gettext('Hide') ?></option>
-            <option value="yes" <?= !empty($pconfig['gatewaysinvert']) ? 'selected="selected"' : '' ?>><?= gettext('Show') ?></option>
-          </select>
-          <select id="gatewaysfilter" name="gatewaysfilter[]" multiple="multiple" class="selectpicker_widget">
+          <select id="gatewaysfilter" name="gatewaysfilter[]" multiple="multiple" class="selectpicker_widget" title="<?= html_safe(gettext('All')) ?>">
 <?php foreach ($gateways as $gwname => $unused): ?>
             <option value="<?= html_safe($gwname) ?>" <?= in_array($gwname, $pconfig['gatewaysfilter']) ? 'selected="selected"' : '' ?>><?= html_safe($gwname) ?></option>
 <?php endforeach;?>
           </select>
-          <button id="submitd" name="submitd" type="submit" class="btn btn-primary" value="yes"><?= gettext('Save') ?></button>
+          <input id="submitd" name="submitd" type="submit" class="btn btn-primary" value="<?=gettext("Save");?>" />
```

Comment:
this is probably not a good idea because the old one guarantees that the value in the post is "yes", now it would be a localized string.